### ISSUE TITLE
Hi from the author of Mantine DataTable!

### ISSUE THREAD

**Issue by icflorescu** [hughfitzgerald/cc-rentals#1]
This is @icflorescu, the author of [Mantine DataTable](https://icflorescu.github.io/mantine-datatable/) open-source component.

I don't think I've got the chance to thank you for sponsoring my project.
I recently discovered you are using Mantine DataTable in this repo.
Would you care to share a bit of feedback with me regarding your experience with it?
I'm trying to understand the users' stories and experience with my component in order to see how/if I could further improve the project.

Also, I'm planning to compile a list of startups / projects that are using Mantine DataTable and add it to the project's documentation website. Would you be ok with me adding a logo/link to [ccrentals](https://www.ccrentals.org/)?

Thanks a lot for using [Mantine DataTable](https://icflorescu.github.io/mantine-datatable/) and sorry for approaching you like this!

**Comment by hughfitzgerald** [comment]
It's great to hear from you! Prior to this project I'd never used Mantine,
React, Node.js, or even JavaScript! When looking at various component
libraries I was looking for an attractive, easy-to-use table with sorting.
I found your component and that got me turned on to Mantine and I never
turned back.

I ran into a bit of trouble in March when I tried to upgrade Mantine to v6
and your component to v2. The problems might've had to do with my own lack
of knowledge in setting up responsive styles, but to be honest I haven't
yet tried upgrading again. If I do, perhaps I'll reach out to you if I
still run into issues.

I wish I had more specific feedback for you. I'm fairly inexperienced,
never did web development, I have a CS degree from 2007 but I've never
worked as a software engineer (I'm now a practicing lawyer). My use case
was pretty simple and your component was just right for it. You are
absolutely welcome to include the logo/link to ccrentals.

Thanks so much for contributing this component to the world, and thanks for
reaching out for my feedback.

Stephen
